dw-icon-button.js

```diff
@@ -206,44 +245,81 @@ export class DwIconButton extends buttonFocus(LitElement) {
      * Later on, it's value will be changed when entry animation is started (__onStart).
      * And when entry animation is completed, that promise gets resolved.
      */
-    this.waitForEntryAnimation = new Promise( (resolve) => {resolve()});
+    this.waitForEntryAnimation = new Promise((resolve) => {
+      resolve();
+    });
     this.__bindActiveEvents();
     this.__bindInactiveEvents();
+
+    this.addEventListener("mousedown", this._onMouseDown);
+    this.addEventListener("blur", this._onBlur);
   }
 
   disconnectedCallback() {
     this.__unbindActiveEvents();
     this.__unbindInactiveEvents();
     super.disconnectedCallback && super.disconnectedCallback();
+
+    this.removeEventListener("mousedown", this._onMouseDown);
+    this.removeEventListener("blur", this._onBlur);
   }
 
   render() {
     return html`
-      <button style=${this._buttonStyle()} 
-        tabindex="${this.disabled ? -1 : ''}" 
-        class="center-center layout vertical">
-        <dw-icon 
-          .name="${this.icon}" 
-          .size=${this.iconSize} 
-          .iconFont="${this.iconFont}"
-          ?disabled="${this.disabled}"></dw-icon>
-        </dw-icon>
-      </button>
-      
-      ${this.title && !isTouchDevice() ? html`
-      <dw-tooltip
-        .trigger=${"mouseenter"}
-        .forEl=${this}
-        .offset=${[0, 8]}
-        .extraOptions=${{ delay: [500, 0] }}
-        .content=${this.title}>
-      </dw-tooltip>
-      ` : ''}
-    `
+      <div id="wrapper" @click=${this._onTooltipContainerClick}>
```

Comment:
@hitendreamworld I don't understand why this is changed?

1. Why do we need to add wrapper?
2. Why do we need to stop propagation of the event when disabled?